### PR TITLE
Update CommandPalette.tsx

### PR DESCRIPTION
**Pull request by Himanxu1**
remove the @ts-ignore comments

Issue Number: #3658 

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by souravjain540** [comment]
@arielweinberger please approve

**Comment by souravjain540** [comment]
@tupe12334 approve this

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useMemo, useContext } from "react";
-// @ts-ignore
-import ReactCommandPalette from "react-command-palette";
-// @ts-ignore
+
+import ReactCommandPalette from "@types/react-command-palette";
```
Comment (tupe12334) [inline packages/amplication-client/src/CommandPalette/CommandPalette.tsx:3]: Why did you change the import to @types/react-command-palette?

**Comment by tupe12334** [comment]
Why didn't you add the @types/react-command-palette lib to the dev dependencies?

**Comment by tupe12334** [comment]
> @types/react-command-palette

Add it to the dev dependencies not the code

**Comment by arielweinberger** [comment]
So I see the package is already in the dev dependencies ([see here](https://github.com/Himanxu1/amplication/blob/patch-7/packages/amplication-client/package.json#L27))

**Review by arielweinberger — CHANGES_REQUESTED** [review]
The correct way to fix this is to install:

```
npm install -D @types/react-command-palette
```

into Amplication Client, and then remove the @ts-ignore line. Will you please try that @Himanxu1 and make sure the client project compiles fine with no TS complaints?

**Comment by yuval-hazaz** [comment]
@arielweinberger what do we need to move forward with this PR?

**Comment by AllMikeNoIke** [comment]
Hi @arielweinberger, I have made pull request #3791 to also address issue #3658. I know @Himanxu1 was originally assigned to this issue, so if this goes against community guidelines, I will close the PR.

**Pull request closed without merging**